proc/proc_test.go

```diff
@@ -495,6 +495,34 @@ func TestSwitchThread(t *testing.T) {
 	})
 }
 
+func TestCGONext(t *testing.T) {
```

Comment:
Yes. This test fails without my patch with exact same error. (with updated cgotest fixture only).
Another cgo test passes (with/without extra C header).